src/test/java/nl/tudelft/ewi/devhub/server/backend/Bootstrapper.java

```diff
@@ -205,6 +214,20 @@ public void prepare(String path) throws IOException, ApiError, GitClientExceptio
 				catch (Exception e) {
 					projects.provisionRepository(groupEntity, members);
 				}
+				
+				Delivery delivery = new Delivery();
```

Comment:
I personally like @LiamClark 's way of doing this more. See the mapper at https://github.com/LiamClark/devhub/commit/2e92648a2749cc2384a43a49290b5751b786769b . Just fetch them all, and group. We're talking about +/- 10 entities anyway.